lib/rules/focus-order-semantics.json

```diff
@@ -0,0 +1,13 @@
+{
+  "id": "focus-order-semantics",
+  "selector": "div, h1, h2, h3, h4, h5, h6, [role=heading], p, span",
+  "matches": "inserted-into-focus-order-matches.js",
+  "tags": ["cat.name-role-value", "cat.keyboard"],
+  "metadata": {
+    "description":
+      "Ensures elements placed in the focus order have an appropriate aria role for interactive content",
+    "help":
+      "Elements placed in the focus order must have an aria role appropriate for interactive content."
+  },
+  "any": ["valid-focus-order-widget", "valid-scrollable-semantics"]
```

Comment:
`all` and `none` are required properties.  Please add them as empty arrays.

Reply:
🙌thank you for catching this, I couldn't figure out why tests wouldn't run